### ISSUE TITLE
Where to put figure source files?

### ISSUE THREAD

**Issue by DocOtak** [cf-convention/cf-conventions#559]
This is an ex post facto issue for #551 as there is an unresolved question.

In #551 the original author (@neumannd) of two of the figures in the CF conventions provided the R source files that generated the figures in the first place. These were added along side the figured they generated in the images directory with the same file name and a different extensions.

In the discussion of that PR we weren't sure if this is where the source files should be kept. We did want to get the source files in now so they wren't lost and could decide where to move them later.

One possibly is to have a src directory in the images directory that has all the generating programs. We could also just keep the source next to the figures as is like they are now.

**Comment by JonathanGregory** [comment]
Thanks, Barna @DocOtak, Antonio @cofinoa and Daniel @neumannd. To me it seems fine and convenient to keep the source files in the [`images`](https://github.com/cf-convention/cf-conventions/tree/main/images) directory of the repo, along with the image files. However, I don't have have a strong opinion. I wonder what any of you, or @davidhassell, think would be best.

**Comment by cofinoa** [comment]
Agreed. Let’s keep source files next to their images by default for discoverability (same basename, different extension). If any CI/build tool complains about non-image files in `images/`, we’ll relocate sources to `images/src/` (or similar) and document the exception.

**Comment by DocOtak** [comment]
Yeah where they are now is fine, and as @davidhassell and @cofinoa say, we can revisit if anything _actually_ breaks.
